modules/oadp-dpt-use-case-bsl-spec.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-data-protection-test.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-dpt-use-case-bsl-spec.adoc_{context}"]
+= Running a data protection test by configuring a backup storage location specification
+
+[role="_abstract"]
+In this use case, you configure the `DataProtectionTest` (DPT) custom resource (CR) by specifying the backup storage location (BSL) specification instead of a BSL name. You then run the DPT CR to verify the Container Storage Initiative (CSI) snapshot readiness and the data upload performance to the storage bucket.
+
+.Prerequisites
+
+* You have logged in to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {oadp-short} Operator.
+* You have created the `DataProtectionApplication` (DPA) CR.
+* You have configured a backup storage location (BSL) to store the backups.
+* You have an application with persistent volume claims (PVCs) running a separate namespace.
+
+
+.Procedure
+
+.  Create a manifest file for the DPT CR and configure the following details as shown in the example:
+.. `backupLocationSpec` field
+.. `csiVolumeSnapshotTestConfigs` object 
+.. `uploadSpeedTestConfig` object
++
+.Example DPT CR
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionTest
+metadata:
+  name: dpt-sample
+  namespace: openshift-adp
+spec:
+  backupLocationSpec: # <1>
+    provider: aws
+    default: true
+    objectStorage:
+      bucket: sample-bucket # <2>
+      prefix: velero
+    config:
+      region: us-east-1 # <3>
+      profile: "default"
+      insecureSkipTLSVerify: "true"
+      s3Url: "https://s3.amazonaws.com/sample-bucket"
+    credential: # <4>
+      name: cloud-credentials
+      key: cloud
+  uploadSpeedTestConfig: # <5>
+    fileSize: 50MB
+    timeout: 120s
+  csiVolumeSnapshotTestConfigs: # <6>
+    - volumeSnapshotSource:
+        persistentVolumeClaimName: mongo
+        persistentVolumeClaimNamespace: mongo-persistent
+      snapshotClassName: csi-snapclass
+      timeout: 2m
+  forceRun: true
+----
+<1> Configure the BSL spec by specifying details such as the cloud provider.
+<2> Specify the bucket name. In this example, the bucket name is `sample-bucket`.
+<3> Specify the cloud provider region.
+<4> Specify the cloud credentials for the storage bucket.
+<5> Configure the `uploadSpeedTestConfig` object by setting the `fileSize` and `timeout` fields.
```

Comment:
I think we should mark it as optional just in case Customer only wants to test FS backup performance or CSI readiness 
```suggestion
<5> (Optional) Configure the `uploadSpeedTestConfig` object by setting the `fileSize` and `timeout` fields.
```